Plots/MapProjections/NCL_lcnative_1_lg.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_lcnative_1_lg.py
+================
+This script illustrates the following concepts:
+    - Drawing contours over a map using a native lat,lon grid
+    - Drawing filled contours over a Lambert Conformal map
+    - Zooming in on a particular area on a Lambert Conformal map
+    - Subsetting a color map
+    - Using all three Cartopy Lambert projections to find the best fit for visualization
+    - Implementing best practices for choosing contour color scheme
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lcnative_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lcnative_1_lg.png
+
+"""
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+
+
+from geocat.viz import cmaps as gvcmaps
+import geocat.datafiles as gdf
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/pre.8912.mon.nc"), decode_times=False)
+
+# Extract a slice of the data
+t = ds.pre.isel(time=0)
+
+###############################################################################
+# Plot:
+
+plt.figure(figsize=(14, 14))
+
+
+def Plot(row, col, pos, proj, title):
+    '''
+    Args:
+        
+        row (:class: 'int'): 
+            number of rows necessary for subplotting of visualizations
+        col (:class: 'int'): 
+            number of columns necessary for subplotting 
+        pos (:class: 'int'): 
+            position of visualization in m x n subplot
+        proj (:class: 'cartopy.crs'): 
+            which projection to visualize
+        title (:class: 'str'): 
+            center title of respective visualization
+    '''    
+    # Generate axes using Cartopy and draw coastlines
+    projection = proj
```

Comment:
Could use `proj` directly throughout this function without creating a new variable  `projection`.